### PR TITLE
feat(Alert): add `fitted` prop

### PR DESCRIPTION
**Pull request by layershifter**
### `fitted` prop

This PR adds a new prop to `Alert` component, it allows to only take up the width of its content.

***

Also adds an example into `Usage` with fixed container width that wraps `Alert`.

### CODE REVIEW

File: docs/src/examples/components/Alert/Usage/AlertExampleWidth.shorthand.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { useRangeKnob } from '@stardust-ui/docs-components'
+import { Alert } from '@stardust-ui/react'
+import * as React from 'react'
+
+const AlertExampleWidth = () => {
+  const [width] = useRangeKnob({ name: 'width', initialValue: '500px' })
```
Comment (lucivpav) [inline docs/src/examples/components/Alert/Usage/AlertExampleWidth.shorthand.tsx:6]: Wouldn't it be better to specify the bounds on knob instead? Let's create a ticket for it?

Comment (layershifter) [reply docs/src/examples/components/Alert/Usage/AlertExampleWidth.shorthand.tsx:6]: I am working on PR that will introduce it

File: packages/react/src/components/Alert/Alert.tsx [hunk 2]
```diff
@@ -59,6 +59,9 @@ export interface AlertProps
   /** Controls Alert's relation to neighboring items. */
   attached?: boolean | 'top' | 'bottom'
 
+  /** An alert can only take up the width of its content. */
+  compact?: boolean
```
Comment (mnajdova) [inline packages/react/src/components/Alert/Alert.tsx]: Why not `fitted`? It feels awkward to have two different terms across the components for the same thing? If we decide that one is better then the other, we can rename all of them at once. What do you think?

File: docs/src/examples/components/Alert/Usage/AlertExampleWidth.shorthand.tsx (hunk 1 above)
Comment (layershifter) [reply docs/src/examples/components/Alert/Usage/AlertExampleWidth.shorthand.tsx:6]: Here it is #1876.

File: packages/react/src/components/Alert/Alert.tsx (hunk 2 above)
Comment (layershifter) [reply packages/react/src/components/Alert/Alert.tsx]: Good catch, `fitted` will work better there and will be consistent 👍

**Review by mnajdova — APPROVED** [review]

File: packages/react/src/themes/teams/components/Alert/alertVariables.ts [hunk 3]
```diff
@@ -55,12 +56,13 @@ export default (siteVars: SiteVariablesPrepared): AlertVariables => {
     color: siteVars.colors.grey[500],
     fontWeight: siteVars.fontWeightRegular,
     minHeight,
-    padding: `0 0 0 ${pxToRem(16)}`,
+    padding: `0 ${pxToRem(16)}`,
```
Comment (layershifter) [inline packages/react/src/themes/teams/components/Alert/alertVariables.ts:59]: @codepretty can you please check this one? Is it correct?

Comment (codepretty) [reply packages/react/src/themes/teams/components/Alert/alertVariables.ts:59]: Looks good to me

**Review by codepretty — APPROVED** [review]

**Pull request merged**